bolt-project.yaml

```diff
@@ -0,0 +1,3 @@
+---
+analytics: false
+name: systemd
```

Comment:
I don't think we need the file? This repo isn't supposed to be a bolt project?

Reply:
I removed it

Reply:
I figured it out. the module was probably cloned as `puppet-systemd`, but the task name is `systemd::systemctl_show`. So the directory needs to ne renamed to `systemd`.

```
$ cp -r puppet-systemd systemd
$ bolt task show systemd::systemctl_show --modulepath=.
systemd::systemctl_show
  Get systemctl show output of a unit

Usage
  bolt task run systemd::systemctl_show --targets <targets> [bin_path=<value>]
  [properties=<value>] unit_name=<value>

Parameters
  bin_path  String[1]
    Path to systemctl binary
    Default: "/usr/bin/systemctl"

  properties  Array[String[1]]
    Properties to retrieve from the unit
    Default: ["ActiveState", "SubState", "LoadState", "UnitFileState"]

  unit_name  String[1]
    Name of the unit

Module
  /home/bastelfreak/code/modulesync_config/modules/voxpupuli/systemd
```